notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-083_m1m3_bump_test_failure_analysis.ipynb

```diff
@@ -502,10 +502,150 @@
     "    return df\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #271.        # Calculate the total number of failures
</pre><p>I think this should say total number of tests</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>